src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -122,6 +123,16 @@ public InstructorAttributes getInstructorForRegistrationKey(String registrationK
         return instructorsDb.getInstructorForRegistrationKey(registrationKey);
     }
 
+    /**
+     * Gets all instructors of a course.
+     */
+    public List<String> getInstructorEmailsForCourse(String courseId) {
+        List<String> instructorReturnList = instructorsDb.getInstructorEmailsForCourse(courseId);
```

Comment:
`ReturnList` as a variable name doesn't value add. Just name it `instructorsForCourse`, or even as simple as `instructors`.